src/audio/copier/copier.c

```diff
@@ -1998,6 +2040,10 @@ static const struct comp_driver comp_copier = {
 		.get_total_data_processed	= copier_get_processed_data,
 		.get_attribute			= copier_get_attribute,
 		.position			= copier_position,
+		.dai_ts_config			= copier_dai_ts_config_op,
```

Comment:
how to handle those interface when we want to make copier use module adapter interface?
these are not standard API for module interface.
Or we don't need convert copier to module adapter interface?

Reply:
ok, what's the suitable time for convert copier to module interface?